python/python/lance/torch/data.py

```diff
@@ -52,17 +51,20 @@ def _to_tensor(
             pa.types.is_floating(arr.type.value_type)
             or pa.types.is_integer(arr.type.value_type)
         ):
-            np_arrs = arr.to_numpy(zero_copy_only=False)
-            np_tensor = np.stack(np_arrs)
-            del np_arrs
-            tensor = torch.tensor(np_tensor)
+            np_tensor = arr.values.to_numpy(zero_copy_only=True).reshape(
```

Comment:
is non chunked array garueenteed here?

Reply:
Our lance rust core does not really have concept of `chunked array`. This is read from `lance.dataset`, i think we wont have chunked table in this case?

Reply:
I'm pretty sure I have see chunked arraies being returned. Let me test

Reply:
You can not make a `FixedSizeListArray` from `ChunkedArray`

```
    def test_convert_chunked_fsl(tmp_path: Path):
        arr = pa.chunked_array([[1.0, 2.0, 3.0], [2.0, 3.0, 4.0]], type=pa.float32())
        print(arr)
>       fsl = pa.FixedSizeListArray.from_arrays(arr, 3)

python/tests/torch_tests/test_data.py:266: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 

>   ???
E   TypeError: Cannot convert pyarrow.lib.ChunkedArray to pyarrow.lib.Array
```